File: MODUL.4/PRAK405_2310817120010_NurHikmah.c
#include <stdio.h>

int main()
{
    int n, kelipatan;

    printf("Input\n");
    scanf("%d %d", &n, &kelipatan);

    int hasil[n];
    int total = 0;

    for (int i = 0; i < n; i++)
    {
        int jumlah = 0;

        for (int j = i; j >= 0; j--)
        {
            jumlah += (j + 1) * kelipatan;
        }

        hasil[i] = jumlah;

        total += jumlah;
    }

    printf("\nOutput\n");
    for (int i = 0; i < n; i++)
    {
        for (int j = i; j >= 0; j--)
        {
            printf("(%d * %d)", j + 1, kelipatan);
            if (j > 0)
            {
                printf(" + ");
            }
        }
        printf(" = %d\n", hasil[i]);
    }

    printf("%d\n", total);

    return 0;
}

File: MODUL.4/PRAK405_2310817120010_NurHikmah.py
n, kelipatan = map(int, input("Input\n").split())

hasil = []
total = 0

for i in range(n):
    jumlah = 0

    for j in range(i, -1, -1):
        jumlah += (j + 1) * kelipatan

    hasil.append(jumlah)
    total += jumlah

print("\nOutput")
for i in range(n):
    for j in range(i, -1, -1):
        print(f"({j + 1} * {kelipatan})", end="")
        if j > 0:
            print(" + ", end="")
    print(f" = {hasil[i]}")

print(total)